src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +570,90 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with Z, insert the rules here
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+              CASE('ZONEHVAC:TERMINALUNIT:VARIABLEREFRIGERANTFLOW')
+                isVariableVolume = .FALSE.
+                CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                nodiff = .false.
+                OutArgs(1:13) = InArgs(1:13)
+                IF (SameString(InArgs(14), 'FAN:VARIABLEVOLUME')) THEN
+                  isVariableVolume = .TRUE.
+                  OutArgs(14) = 'Fan:SystemModel'
+                  OutArgs(15) = TRIM(InArgs(15))
+                  sysFanName = TRIM(InArgs(15))
+                ELSE
+                  OutArgs(14:15) = InArgs(14:15)
+                ENDIF
+                OutArgs(16:CurArgs) = InArgs(16:CurArgs)
+                CALL WriteOutIDFLines(DifLfn, 'ZoneHVAC:TerminalUnit:VariableRefrigerantFlow', CurArgs, OutArgs, NwFldNames, NwFldUnits)
+
+                IF (isVariableVolume) THEN
+                  ! create fan system model object
+                  ObjectName = 'Fan:SystemModel'
+                  DO Num3 = 1, NumFanVariableVolume
+                    IF (SameString(OldFanVO(Num3)%oldFanName, sysFanName)) THEN
+                      CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                      OutArgs(1) = TRIM(sysFanName)
+                      OutArgs(2) = OldFanVO(Num3)%availSchedule
+                      OutArgs(3) = OldFanVO(Num3)%inletAirNodeName
+                      OutArgs(4) = OldFanVO(Num3)%outletAirNodeName
+                      OutArgs(5) = OldFanVO(Num3)%maxAirFlow_str
+                      OutArgs(6) = 'Continuous'                             !- Speed Control Method
+                      IF (SameString(OldFanVO(Num3)%minFlowInputMethod, "FixedFlowRate")) THEN
+                        IF (.NOT. SameString(OldFanVO(Num3)%maxAirFlow_str, "AUTOSIZE")) THEN
+                          READ(OldFanVO(Num3)%fanPowerMinAirFlow_str, '(F15.5)') fanPowerMinAirFlow
```

Comment:
Nitpicking here, but it's better to use `ProcessNumber` instead of `READ` to protect against an incoming values that isn't really a number and could cause transition to crash. See example usage here:
https://github.com/NREL/EnergyPlus/blob/342372be71855580101fbeebcd3696bb94961b51/src/Transition/CreateNewIDFUsingRulesV8_8_0.f90#L980-L982

Reply:
Thanks. I just changed the `READ` to `ProcessNumber`. I'll take a note of this function and remember to use it next time.